features/cellular/framework/AT/AT_CellularNetwork.h

```diff
@@ -180,6 +178,8 @@ class AT_CellularNetwork : public CellularNetwork, public AT_CellularBase
     AuthenticationType _authentication_type;
     int _cell_id;
     nsapi_connection_status_t _connect_status;
+    virtual nsapi_error_t do_user_authentication();
```

Comment:
No need for error value anymore

Reply:
Do you mean no error return value? What about NSAPI_ERROR_AUTH_FAILURE?

Reply:
I think this topic is not about adding target, so I didn’t make any change about this. Please avoid this topic in this PR.